guides/profiles/63-invoiceresponse/process/use-cases/use-cases.adoc

```diff
@@ -209,3 +209,18 @@ Service provider +
 Seller
 |Result |Seller is notified that an Invoice has been accepted and will be paid on due date.
 |====
+
+[[uc9]]
+.Use case 9 — Invoice is partially paid.
+[cols="1h,5",options="header"]
+|====
+|Use Case number |9
+|Use Case Name |Partial payment.
+|Assumption and description |The Buyer previously accepted the invoice but needs to pay it in more than one payments, either by previous agreement or for another reason. The invoice will be paid fully as accepted.
+|The flow |Invoice response is send with Status "Paid" and Clarification Reasons as PPD (Partially paid).
```

Comment:
Status PAID is code PD? This should be clearly stated and tested against. (Eg. is it allowed to have PPD Clarification Reason with status code other than PD?)

Reply:
Changed as proposed